common/autoware_universe_utils/README.md

```diff
@@ -49,56 +49,83 @@ explicit TimeKeeper(Reporters... reporters);
   - Ends tracking the processing time of a function.
   - `func_name`: Name of the function to end tracking.
 
+##### Note
+
+- It's possible to start and end time measurements using `start_track` and `end_track` as shown below:
+
+  ```cpp
+  time_keeper.start_track("example_function");
+  // Your function code here
+  time_keeper.end_track("example_function");
+  ```
+
+- For safety and to ensure proper tracking, it is recommended to use `ScopedTimeTrack`.
+
 ##### Example
 
 ```cpp
 #include "autoware/universe_utils/system/time_keeper.hpp"
 
 #include <rclcpp/rclcpp.hpp>
 
+#include <std_msgs/msg/string.hpp>
+
+#include <chrono>
 #include <iostream>
 #include <memory>
+#include <thread>
+
+class TimeKeeperExample : public rclcpp::Node
```

Comment:
ExampleNode sounds better since the word `TimeKeeper` is everywhere currently and searchability is not so high.

Reply:
fixed!